client/components/public/FAQ.jsx

```diff
@@ -184,15 +234,22 @@ FAQ = class FAQ extends Component {
             <br /><br />
             We recommend 4-6 people on a team. It can be an advantage to divvy up the work (cutting, constructing, googling, etc.).
           </Accordion.Content>
-
+          <Accordion.Title active={activeIndex === 17} index={17} onClick={(e,p) => this.handleClick(e,p)} >
+            <Icon color="yellow" name="dropdown" />
+            <Icon name="user"/>
+            Don't Have a team?
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 17}>
+              First create an account. Teams looking for members here: (<a href="#">TBA</a>)
```

Comment:
Can we add "view"? eg
"*View* Teams looking for members"

Reply:
Fixed!